spec/requests/collections_spec.rb

```diff
@@ -42,7 +42,8 @@
             description: 'This is a very good collection.',
             contact_email: user.email,
             access: 'world',
-            managers: user.email
+            managers: user.email,
+            depositor_sunets: 'maya.aguirre,jcairns, cchavez, premad, giancarlo, zhengyi'
```

Comment:
fwiw, the spacing after the first comma should be a single space, not no space, eh?

Reply:
I was testing that it can handle with or without a space.